apps/backend/src/api/vms.py

```
@@ -35,19 +38,19 @@ async def get_vm_logs(
             db_session_factory=session_factory,
             ssh_client=ssh_client
         )
-        
+
         # Get device ID from hostname
         async with session_factory() as db:
             device_service = DeviceService(db)
             device = await device_service.get_device_by_hostname(hostname)
-            device_id = device.id
-        
+            device_id = cast(UUID, device.id)
```

Ruff (0.12.2)

Add quotes to type expression in `typing.cast()`

Add quotes

(TC006)